public/app/plugins/datasource/prometheus/querybuilder/components/QueryPreview.tsx

```diff
@@ -18,15 +18,17 @@ export function QueryPreview({ query }: Props) {
   const hightlighted = Prism.highlight(promQueryModeller.renderQuery(query), promqlGrammar, 'promql');
 
   return (
-    <EditorFieldGroup>
-      <EditorField label="Query text">
-        <div
-          className={cx(styles.editorField, 'prism-syntax-highlight')}
-          aria-label="selector"
-          dangerouslySetInnerHTML={{ __html: hightlighted }}
-        />
-      </EditorField>
-    </EditorFieldGroup>
+    <EditorRow>
+      <EditorFieldGroup>
+        <EditorField label="Query text">
```

Comment:
```suggestion
        <EditorField label="Query preview">
```
The same as above, shouldn't we unify the name in toggle (Preview -> Query preview) with label (Query text -> Query preview)?

Reply:
yes, totally agree :+1: